### ISSUE TITLE
Editing cell context menu wroung offset

### ISSUE THREAD

**Issue by Paraiso83** [bpmn-io/dmn-js#359]
When editing a input cell, by clicking on it or right clicking, the pencil icon and the context menu are placed  far from the cell, if the table is included inside a DIV on the middle of a page.

Steps to reproduce:
- Have a Decision Table on a DIV in the middle of a page;
- Scroll the page down;
- Right mouse click on the cell;

![image](https://user-images.githubusercontent.com/6715233/41528692-aa9595fe-72e2-11e8-8a6b-69ade2255f1d.png)
![image](https://user-images.githubusercontent.com/6715233/41528836-286b7fe8-72e3-11e8-8a94-79e733012eb0.png)

Tested it also on camunda simulator, that has similar implementation, and have same result.

**Comment by nikku** [comment]
Did you apply `position: relative` to the DMN editor parent element?

**Comment by Paraiso83** [comment]
Yes, the having `position: relative` on the parent elements (all of the parents until the body) doesn't change the behavior
